src/Parameters/PublishRecordingsParameters.php

```diff
@@ -64,7 +64,7 @@ public function getHTTPQuery(): string
         return $this->buildHTTPQuery(
             [
                 'recordID' => $this->recordingId,
-                'publish'  => !is_null($this->publish) ? ($this->publish ? 'true' : 'false') : $this->publish,
+                'publish'  => $this->publish ? 'true' : 'false',
```

Comment:
It should be `'publish' => $this->publish ?? 'false' ? 'true' : 'false',` which is similar to the previous implementation because the constructor is not preventing null value of a boolean

Reply:
To double check.

Reply:
should be good!